packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBarRow.test.js

```diff
@@ -63,24 +61,24 @@ describe('<SpanBarRow>', () => {
   beforeEach(() => {
     props.onDetailToggled.mockReset();
     props.onChildrenToggled.mockReset();
-    wrapper = mount(<SpanBarRow {...props} />);
+    render(<SpanBarRow {...props} />);
   });
 
   it('renders without exploding', () => {
-    expect(wrapper).toBeDefined();
+    expect(screen).toBeDefined();
```

Comment:
this test will then be something like:
```suggestion
    expect(() => render(<SpanBarRow {...props} />)).not.toThrow();
```
```